test/runtests.jl

```diff
@@ -0,0 +1,17 @@
+using ChainRulesCore
+using ChainRulesTestUtils
+using Random
+using Test
+
+@testset "ChainRulesTestUtils.jl" begin
+    double(x) = 2x
+    @scalar_rule(double(x), 2)
+    test_scalar(double, 2)
+
+    fst(x, y) = x
+    ChainRulesCore.frule(::typeof(fst), x, y, _, dx, dy) = (x, dx)
+    ChainRulesCore.rrule(::typeof(fst), x, y) = fst_pullback(dout) = (NO_FIELDS, @thunk(identity(dout)), Zero())
+
+    frule_test(fst, (2, 4.0), (3, 5.0))
+    rrule_test(fst, (2, 4.0), (3, 5.0))
```

Comment:
Forgot to change this in response to changing the internals:
```suggestion
    rrule_test(fst, 2.5, 2.0, 3.0)
```